github/Contribution.md

```diff
@@ -0,0 +1,98 @@
+
+## Contribute to the documentation
+
+We are constantly looking for ways to make the documentation clearer and more accurate.
+Please let us know how the documentation can be improved such as typos and any content that is missing, unclear or inaccurate. 
+We will be happy to review your suggestion or to make changes!
+
+## Create a Pull Request
+
+We strongly suggest you to check the existing PRs or issues before developing any code to make sure no one else is working on the same issue already.
+If you are unsure, it is a good idea to comment on the related issue to check if someone is already on it. 
+
+Basic proficiency with `git` is necessary to contribute to the library and the code requires Python 3.9.
+Follow the steps below to start contributing:
+
+  * Fork the repository with the **Fork** button on the repository page.
+  * Clone your fork locally, and add the base repository as remote:
+
+  ```shell
+  git clone <URL to your fork>
+  cd <local fork directory>
+  git remote add upstream <URL of original repository>
+  ``` 
+  
+  * Create a new branch to commit your development changes. **Do not work on the main branch**.
+
+  ```shell
+  git checkout -b descriptive-branch-name
+  ```
+
+  * Create a development environment as detailed [here](https://github.com/iai-group/guidelines/blob/main/python/README.md#local-development-configuration).
+  * Develop on your branch.  
+
+    - As you work on the code, you can run the tests under `tests` with `pytest tests -vv`. New code features should be accompanied by tests in order to be merged into the main branch, unless a valid reason for the absence of tests is provided in the issue description.  
+    - If you work on the documentation, you can build it locally with this command: `sphinx-multiversion docs/source build/html`. Make sure that the required packages are installed before running the command. This command will create the documentation locally under `build/html`.
+  
+  * Once the development is finished:
+  
+    - The modified files can be added with `git add` and committed with `git commit`. Remember to write a [good commit message](Git_commit.md).
+    - To keep your copy of the code up to date with the original repository, rebase your branch on upstream/branch before you open a pull request:
+
+    ```shell
+    git fetch upstream
+    git rebase upstream/main
+    ```
+
+    - Push you changes: `git push -u origin descriptive-branch-name`. If a PR is already open use the `--force` flag.
+
+  * To open a PR, go to your forked repository on GitHub and click on **Pull Request**. Make sure to describe what the PR changes and mention the related issue in the PR description. More information on the code review [here](Code_review.md).
+
+
+## Conventions
+
+We follow the [IAI Python Style Guide](https://github.com/iai-group/styleguide/tree/main/python), and all contributions are subject to the guidelines described therein.
```

Comment:
Nit: Add an empty line at the bottom. (Files should end with a single newline character)